cvat-ui/src/components/layout-grid/LayoutGrid.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import './styles.scss';
+
+export default function LayoutGrid(): React.ReactPortal | null {
+    const [showGrid, setShowGrid] = useState(false);
+
+    const listener = (e: KeyboardEvent): void => {
+        if (e.key === 'g' && e.ctrlKey) {
+            setShowGrid(!showGrid);
+        }
+    };
+
+    useEffect(() => {
+        window.addEventListener('keyup', listener);
+        return () => {
+            window.removeEventListener('keyup', listener);
+        };
+    }, [showGrid]);
```

Comment:
Could we setup listeners using currently integrated library?  
For example look at using ``GlobalHotKeys`` in ``cvat-app.tsx``. Can we add <GlobalHotKeys /> to portalContent?

Reply:
fixed